### PR TITLE
add -W argument to ignore warnings

### PR DESCRIPTION
**Pull request by Randomneo**
sometimes you'd like to ignore warnings but there is no argument for it. WDYT

### CODE REVIEW

**Comment by wbolster** [comment]
not opposed to the idea itself, but what should this do?

```
  --disable-warnings, --disable-pytest-warnings
                        Disable warnings summary
```

or

```
pytest-warnings:
  -W PYTHONWARNINGS, --pythonwarnings=PYTHONWARNINGS
                        Set which warnings to report, see -W option of Python
                        itself
```
?

also, ‘disable’ has a typo

**Comment by Randomneo** [comment]
fixed typos. I assume it should do the second one `-W PYTHONWARNINGS`.

Should I change description from `disable warnings` to `ignore warnings`?

Also changed arg to make it more obvious

File: python-pytest.el [hunk 1]
```diff
@@ -125,6 +125,7 @@ When non-nil only ‘test_foo()’ will match, and nothing else."
    [("-c" "color" "--color")
     ("-q" "quiet" "--quiet")
     ("-s" "no output capture" "--capture=no")
+    ("-w" "disable warnings" "-W ignore")
```
Comment (wbolster) [inline python-pytest.el:128]: let's keep it `-W` (capital) so that it matches both the corresponding and `pytest` and `python` args (both also use `-W`)

Comment (wbolster) [reply python-pytest.el:128]:
```suggestion
    ("-W" "ignore warnings" "-W ignore")
```

**Review by wbolster — APPROVED** [review]

File: python-pytest.el [hunk 2]
```diff
@@ -125,6 +125,7 @@ When non-nil only ‘test_foo()’ will match, and nothing else."
    [("-c" "color" "--color")
     ("-q" "quiet" "--quiet")
     ("-s" "no output capture" "--capture=no")
+    ("-W" "ignore warnings" "-W ignore")
     (python-pytest:-v)]]
```
Comment (wbolster) [inline python-pytest.el:129]: let's keep alphabetical ordering btw

```suggestion
    (python-pytest:-v)
    ("-W" "ignore warnings" "-W ignore")]]
```

**Comment by wbolster** [comment]
thanks, merged with a few tiny tweaks! 🚀

**Pull request merged**